WcaOnRails/spec/controllers/api_competitions_controller_spec.rb

```diff
@@ -264,8 +264,12 @@
         expect(response.status).to eq 200
         parsed_body = JSON.parse(response.body)
         expect(parsed_body).to eq(
-          "formatVersion" => "1.0",
+          "format_version" => "1.0",
```

Comment:
The WCIF spec currently says `formatVersion`, not `format_version`. Can we stick with camelCase for now, and switch everything to snake_case in a later PR?